promql/value.go

```diff
@@ -168,6 +168,21 @@ func (p HPoint) MarshalJSON() ([]byte, error) {
 	return json.Marshal([...]interface{}{float64(p.T) / 1000, h})
 }
 
+// histogramSamples returns the number of samples representing the histogram.
+// each sample is 4 bytes.
+func (p HPoint) histogramSamples() int {
```

Comment:
I would call it more like `histogramSize` and explain it as "measured in relation to a conventional `FPoint` i.e. 16 bytes.